src/components/headerlinks.jsx

```diff
@@ -1,17 +1,16 @@
 import { NavLink } from 'react-router-dom';
-import { FaTrash } from 'react-icons/fa';
 import '../styles/header.css';
 import PropTypes from 'prop-types';
 
 const Headerlink = ({ links }) => (
   <section className="headercontainer parent">
     <div className="headercontainer parentcontainer">
-      <h1>Bookstore CMS</h1>
+      <h1 className="blued">Bookstore CMS</h1>
       <ul className="headercontainer child">
-        {links.map((link) => <li key={link.link}><NavLink className="links" to={link.path}>{link.link}</NavLink></li>)}
+        {links.map((link) => <li key={link.link}><NavLink className="links route" to={link.path}>{link.link}</NavLink></li>)}
       </ul>
     </div>
-    <FaTrash />
+    <a className="fa-solid fa-user blued" href="https://github.com/David-Lanzz/David-Lanzz" />
   </section>
```

Comment:
Template
![image](https://user-images.githubusercontent.com/78251927/230717983-2a104393-4e36-4426-a77a-e44829f17647.png)

Your design 
![image](https://user-images.githubusercontent.com/78251927/230718079-4d8bb238-2d93-4a01-bc62-c874c17a6259.png)

Please kindly add the white `background-color` to the navbar so that your design matches the one provided on [Zeplin](https://zeplin.io/) 🤝

* According to the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md), our final design should match zeplin's design (this includes all elements + the same styling) 😎